app/src/oculusvr/cpp/DeviceDelegateOculusVR.cpp

```diff
@@ -610,6 +610,8 @@ struct DeviceDelegateOculusVR::State {
   ovrJava java = {};
   ovrMobile* ovr = nullptr;
   OculusEyeSwapChainPtr eyeSwapChains[VRAPI_EYE_COUNT];
+  const static uint32_t EYE_SURFACE_COUNT = 2;
```

Comment:
nit: you can use exiting `VRAPI_EYE_COUNT` constant

Reply:
I would like to keep this because it means we are doing doubled-buffer surface instead two buffers each for one eye.